js/classic/ui.js

```diff
@@ -10,42 +10,158 @@
   var ui = exports.classic.ui;
 
   /**
-   * An UI component represents a switch.
+   * Creates a DOM element and wraps it by a jquery selector.
    *
-   * It supports turing on/off operations.
+   * @param {String} nodeName - Name of the DOM element to be created.
+   * @param {String=} className - Additional classes to add if needs.
```

Comment:
just use `String`, and  add description in the back says that it could sometimes be `undefined`